applications/nrf5340_audio/README.rst

```diff
@@ -778,7 +780,12 @@ Selecting the BIS mode
 ======================
 
 The CIS mode is the default operating mode for the application.
-You can switch to the BIS mode by adding the ``CONFIG_TRANSPORT_BIS`` Kconfig option set to ``y``  to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.
+You can switch to the BIS mode by adding the ``CONFIG_TRANSPORT_BIS`` Kconfig option set to ``y`` to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.
```

Comment:
This was written before but, in general, should we rather ask users to edit the Kconfig? With this approach, it may happen that they develop and test with debug and get a completely different build on release.

Reply:
I am pretty sure that users shall edit .conf files and not Kconfig files.
Kconfig files are meant to be static and they define configs and give the configs default values. If an user must change these default values, this shall be done in .conf file(s).
I think having these prj.conf and prj_release.conf files are the "default way in NCS".
As long as the same configs are added to both config files (meaning copying them over when switching from debug to release), the functionality should stay the same.